02LessonTwo/homework/index.html

```diff
@@ -1,10 +1,26 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title></title>
-  <link rel="stylesheet" href="css/style.css">
+  <title>First Site</title>
+  <link rel="stylesheet" href="css/style.css" type="text/css">
 </head>
 <body>
 
+    <h1>Goodbye, space!</h1>
+
+    <div>
+      <img src="http://cdn.theatlantic.com/assets/media/img/photo/2015/11/images-from-the-2016-sony-world-pho/s01_130921474920553591/main_900.jpg?1448476701" alt="Jumping Hamster"
```

Comment:
Fantastic! Loving the use of the alt tag. It looks like you have a great grasp of the basics. Let's try pushing a little more, like using ALL the tags listed in Homework. Great layout; legible and logical. Nice!